mycroft/skills/skill_manager.py

```diff
@@ -172,36 +233,36 @@ def _load_on_startup(self):
 
     def _reload_modified_skills(self):
         """Handle reload of recently changed skill(s)"""
-        reload_occured = False
         for skill_dir in self._get_skill_directories():
             try:
                 skill_loader = self.skill_loaders.get(skill_dir)
                 if skill_loader is not None and skill_loader.reload_needed():
                     skill_loader.reload()
-                    reload_occured = True
+                    if skill_loader.instance:
```

Comment:
use skill_loader.loaded instead.

Reply:
Used the return value of the reload method directly instead.